docs/api.md

```diff
@@ -1,3 +1,10 @@
+---
```

Comment:
I know it would be updated automatically after merge, but pushed it to show how it looks like now